src/scripts/themed_posts.js

```diff
@@ -0,0 +1,64 @@
+import { buildStyle, filterPostElements, blogViewSelector } from '../util/interface.js';
+import { getPreferences } from '../util/preferences.js';
+import { onNewPosts } from '../util/mutations.js';
+import { timelineObject } from '../util/react_props.js';
+
+const styleElement = buildStyle();
+const blogs = new Set();
+let blacklist;
+
+const defaultBackgroundColor = '#FAFAFA';
+
+const hexToRGB = (hex) => {
+  const chars = hex.replace('#', '').split('');
+  const splitnum = chars.length > 3 ? 2 : 1;
+  const arr = [];
+  while (chars.length) {
+    arr.push(parseInt(chars.splice(0, splitnum).join(''), 16));
+  }
+  return arr.join(', ');
+};
+
+const processPosts = async function (postElements) {
+  filterPostElements(postElements, { includeFiltered: true }).forEach(async postElement => {
+    if (postElement.matches(blogViewSelector)) return;
+
+    const { blog: { name, theme: { backgroundColor, titleColor } } } =
+      await timelineObject(postElement);
+
+    if (blacklist.includes(name)) return;
+
+    let backgroundColorHex = hexToRGB(backgroundColor);
+    const titleColorHex = hexToRGB(titleColor);
```

Comment:
\>output of `hexToRGB`
\>name implies they're still hex
:thinking:

```suggestion
    let backgroundColorRGB = hexToRGB(backgroundColor);
    const titleColorRGB = hexToRGB(titleColor);
```

Reply:
...huh, I have no idea what I was thinking there :D